functions/java-jre.bash

```diff
@@ -118,30 +118,30 @@ java_install_or_update() {
 ##    java_zulu_prerequisite(String arch)
 ##
 java_zulu_prerequisite() {
-  echo -n "$(timestamp) [openHABian] Installing Java Zulu prerequisites (libc)... "
+  echo -n "$(timestamp) [openHABian] Installing Java Zulu prerequisites (libc, libstdc++, zlib1g)... "
   if [ "$1" == "Zulu8-64" ] || [ "$1" == "Zulu11-64" ]; then
     if is_aarch64 && [ "$(getconf LONG_BIT)" == "64" ]; then
-      if dpkg -s 'libc6:arm64' 'libncurses5:arm64' 'libstdc++6:arm64' > /dev/null 2>&1; then echo "OK"; return 0; fi
+      if dpkg -s 'libc6:arm64' 'libstdc++6:arm64' 'zlib1g:arm64' > /dev/null 2>&1; then echo "OK"; return 0; fi
```

Comment:
Is it needed for ARM, too ? If yes why did that work so far without ?

Reply:
Yes, it was because it just was not preinstalled on the particular system he used, better safe then sorry.